ext/semian/semian_resource_alloc.c

```diff
@@ -0,0 +1,12 @@
+#include <semian_resource_alloc.h>
+
+const rb_data_type_t
```

Comment:
This could not be inlined because it's not a function, and defining a variable in a header file would have resulted in multiples definitions of the symbol. That is the only reason this is in its own C file.